sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -79,8 +79,12 @@
 PYTORCHJOB_PLURAL = "pytorchjobs"
 PYTORCHJOB_CONTAINER = "pytorch"
 PYTORCHJOB_REPLICA_TYPES = (REPLICA_TYPE_MASTER.lower(), REPLICA_TYPE_WORKER.lower())
-
 PYTORCHJOB_BASE_IMAGE = "docker.io/pytorch/pytorch:1.12.1-cuda11.3-cudnn8-runtime"
+STORAGE_CONTAINER = "storage"
```

Comment:
Let's keep name consistent ?
```suggestion
STORAGE_CONTAINER = "storage-initializer"
```